test/fixtures/sem-html5-scenarios/paragraph-with-id-roles.adoc

```diff
@@ -0,0 +1,2 @@
+[#exciting.lead.awesome]
```

Comment:
I wonder if we should deduplicate roles.
With the current implementation, if I define the following:

`[#exciting.lead.paragraph]` 

Then `paragraph` will be present twice:

```html
<p id="exciting" class="paragraph lead paragraph">
```

Reply:
This gets back to whether we should be adding an implicit role in the first place. But even if we do, I don't think we should worry about it since it's not invalid HTML.